azext_iot/pnp/common.py

```diff
@@ -0,0 +1,60 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+"""
+shared: Define shared data types(enums)
+
+"""
+
+from enum import Enum
+
+
+class RoleResourceType(Enum):
+    """
+    Type of the Resource
+    """
+
+    model = "Model"
+    tenant = "Tenant"
+
+
+class RoleIdentifier(Enum):
+    """
+    Role Identifier
+    """
+
+    modelsPublisher = "ModelsPublisher"
```

Comment:
nit - I think its more pythonic to use `_` than camel case

Reply:
These ones are user facing -- this enum maps to model repository roles.